### PR TITLE
feat(DYN-5976): add forward delete functionality

### PR DESCRIPTION
**Pull request by Enzo707**
forward-delete

### Purpose
This PR implements the forward delete functionality as webview2 doesn't support Accelerator keys
[DYN-5976](https://jira.autodesk.com/browse/DYN-5976)

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB 

### Release Notes

Implements forward delete functionality for searchBar input in librariejs and the mechanism in dynamo to "binding" accelerator keys that are not supported in webview2.
In other words we track accelerator keys in Dynamo and use javascript dispatchEvent method to trigger the librariejs event that execute the forward delete method. 

### Reviewers
@QilongTang 

### FYIs
@RobertGlobant20

### CODE REVIEW

File: src/LibraryViewExtensionWebView2/LibraryViewController.cs [hunk 1]
```diff
@@ -347,6 +348,7 @@ private void Browser_CoreWebView2InitializationCompleted(object sender, CoreWebV
             //The default value of the zoom factor is 1.0. The value that comes from the slider is in percentage, so we divide by 100 to be equivalent
             browser.ZoomFactor = (double)dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale / 100;
             browser.ZoomFactorChanged += Browser_ZoomFactorChanged;
+            browser.KeyDown += Browser_KeyDown;
```
Comment (QilongTang) [inline src/LibraryViewExtensionWebView2/LibraryViewController.cs:351]: Please unsubscribe at `protected void Dispose(bool disposing)` function

**Comment by QilongTang** [comment]
@Enzo707 Can you take another look at https://github.com/DynamoDS/librarie.js/pull/185?

**Review by QilongTang — APPROVED** [review]
